### DISCUSSION TITLE
[select] If "items" is a list of objects and the "label" is not named "label" selected value should show the display value

### DISCUSSION THREAD

**Discussion by JordanDDisch** [chakra-ui/ark#2796] [Q&A] [answered]
if we pass the select items prop an array of objects like:
```json
[
  {
      "name": "Africa",
      "value": "AF"
  },
  {
      "name": "Americas",
      "value": "AM"
  }
]
```

The selected item shows the value as the display value i.e. "AM" instead of "Americas". But if I change the "name" key to "label" it works. Is this configurable?

Example: https://stackblitz.com/edit/github-ku6ba6-zja3vz?file=src%2Froutes%2Findex.tsx

**Comment by segunadebayo** [comment, marked as answer]
Yes, it's configurable. Pass `itemToString` prop to the select component

```tsx
<Select.Root items={regions} name="fw" itemToString={(item) => item.name}>
```
